src/pages/tips-til-jobbsoknaden.jsx

```diff
@@ -82,14 +77,16 @@ export default function TipsTilJobbsoknaden() {
                         <BodyLong>Søknaden må se ryddig ut og bør maksimalt være på en side.</BodyLong>
                     </li>
                     <li>
-                        <BodyLong>Fokuser på hva du kan.</BodyLong>
+                        <BodyLong>
+                            Søknaden må se ryddig ut, uten skrivefeil og bør helst ikke være lenger enn en side.
+                        </BodyLong>
                     </li>
                     <li>
-                        <BodyLong>Unngå skrivefeil.</BodyLong>
+                        <BodyLong>Fokusèr på hva du kan og begrunn med eksempler.</BodyLong>
```

Comment:
Lurer på om aksenttegn på fokuser egentlig skal være `fokusér` og ikke `fokusèr`. Men språkrådet sier at man kan/skal kutte aksenttegn i imperativformer av, og at man bare kan skrive `fokuser`. https://www.sprakradet.no/Vi-og-vart/Publikasjoner/Statssprak/statssprak-32016/aksenttegn-i-norsk

Reply:
Uff, for avansert norsk for meg, jeg spør Rune

Reply:
Rune bekreftet at det skal være `Fokusér`

Reply:
oppdatert, merger